src/main/java/se/kth/meta/entity/Tables.java

```diff
@@ -73,6 +73,9 @@
           cascade = CascadeType.ALL) //cascade type all updates the child entities
   private List<Fields> fields;
 
+  @Transient
+  private int inodeid;
```

Comment:
What is this field (inodeid) doing here? An entity should reflect what's persisted in the DB. Attaching a single inode to a table does not make sense.